inst/@sym/mat_rclist_access.m

```diff
@@ -53,7 +53,7 @@
 % To run these in the test suite, you could move this mfile up to @sym.
 % However, note these are generally tested elsewhere indirectly.
 
-%!test
-%! B = [1 2 3; 5 6 7];
-%! A = sym(B);
-%! assert (isequal (mat_rclist_access(A,[1 2],[2 3]), [B(1,2); B(2,3)]))
+%%!test
+%%! B = [1 2 3; 5 6 7];
+%%! A = sym(B);
+%%! assert (isequal (mat_rclist_access(A,[1 2],[2 3]), [B(1,2); B(2,3)]))
```

Comment:
Yes, I think this is unavoidable if we move them out of `private/`.  This is b/c runtests and doctest are currently based on directory listings not `methods` calls.